mmv1/templates/terraform/examples/cloud_run_service_secure_services.tf.erb

```diff
@@ -0,0 +1,96 @@
+# [START cloud_run_secure_services_backend]
+resource "google_cloud_run_service" "<%= ctx[:vars]['renderer'] %>" {
```

Comment:
```suggestion
resource "google_cloud_run_service" "renderer" {
```

renderer shouldn't need to be a variable. This is simply a unique identifier for the resource for terraforms.

Reply:
Thanks, I've taken this and other comments to mean I shouldn't use the template variable in Terraform resource names, only in Cloud resource names. I've pushed a change with that adjustment.